### DISCUSSION TITLE
SPANISH LANGUAGE IN UI

### DISCUSSION THREAD

**Discussion by wariomx** [WalletConnect/walletconnect-monorepo#3086] [Developer Support] [answered]
How can i Change the language displayed in the modal's UI to spanish? 

I tried this: 
` const wagmiConfig = createConfig({
    autoConnect: true,
    connectors: w3mConnectors({ projectId, chains }),
    publicClient,
    locale: 'es',
  });`

but does not works,

**Comment by finessevanes** [comment, marked as answer]
@wolffhack is doesn't appear `locale` is available in wagmi according to their docs: https://wagmi.sh/react/config

**Reply by wariomx** [reply]
@finessevanes nothing related to language appears in docs.   :(
